<?xml version="1.0"?>
<launch>
	<!--<node pkg="mammoth_snowplow" type="mammoth_waypoint_publisher.py" name="mammoth_waypoint_publisher" output="screen">
		<param name="filename" value="$(find mammoth_snowplow)/config/waypoints/highbay.csv"/>
		<param name="filename" value="$(find mammoth_snowplow)/config/waypoints/single-I.csv"/>
		
		<param name="filename" value="$(find mammoth_snowplow)/config/waypoints/double-I.csv"/>
	</node>-->
	<node pkg="mammoth_snowplow" type="mammoth_snowplow_node" name="mammoth_snowplow_node" output="screen">
		<param name="filename" value="$(find mammoth_snowplow)/config/waypoints/single-I.csv"/>
		<!--<param name="filename" value="$(find mammoth_snowplow)/config/waypoints/double-I.csv"/>-->
		<param name="tolerance_x" value="0.2"/>
		<param name="tolerance_y" value="0.2"/>
		<param name="tolerance_qz" value="0.2"/>
		<param name="tolerance_qw" value="0.2"/>
	</node>
	<node pkg="rostopic" type="rostopic" name="rostopic" args="pub 
	-1 /yeti/state/system std_msgs/String &quot; data: &apos;ready&apos; &quot; "/>
	<node pkg="mammoth_snowplow" type="redirect_node" name="redirect_node" output="screen"/>
	<node pkg="mammoth_snowplow" type="auditory_feedback.py" name="auditory_feedback" output="screen"/>
	<node pkg="mammoth_snowplow" type="fake_laser_node" name="fake_laser_node">
		<param name="border_offset" value="0.7"/>
	</node>
	<node pkg="mammoth_snowplow" type="transform_publisher_node" name="transform_publisher_node"/>
</launch>
